tests/Factory/Psr17FactoryTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Nyholm\Psr7\Factory;
+
+use Nyholm\Psr7\Factory\Psr17Factory;
+use PHPUnit\Framework\TestCase;
+
+class Psr17FactoryTest extends TestCase
+{
+    public function testCreateResponse()
+    {
+        $factory = new Psr17Factory();
+        $r = $factory->createResponse(200);
+        $this->assertEquals('OK', $r->getReasonPhrase());
```

Comment:
Should this be extended to also include a test for a code that doesn’t have a predefined reason phrase? `567` or something otherwise weird? Or can we ignore that because we test that case in ResponseTest.php already? (`ResponseTest::testCanConstructWithUndefinedStatusCode`)

Reply:
Lets make the same test for the factory. 
Thank you